shared/devices/device-revoke/container.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import DeviceRevoke from './'
+import {compose, mapProps} from 'recompose'
+import {connect} from 'react-redux'
+import {isMobile} from '../../constants/platform'
+import {navigateUp} from '../../actions/route-tree'
+import {revoke} from '../../actions/devices'
+
+import type {TypedState} from '../../constants/reducer'
+
+const mapStateToProps = (state: TypedState, {routeProps}) => ({
+  // $FlowIssue getIn
+  device: state.entities.getIn(['devices', routeProps.deviceID]),
```

Comment:
annoying that flow doesn't understand these helpers. maybe we can enhance the flow-typed?